### DISCUSSION TITLE
youku l1c 路由器无法安装

### DISCUSSION THREAD

**Discussion by zheng7fu2** [messense/aliyundrive-webdav#263] [Q&A] [answered]
### 问题描述

在youku l1c 路由器上无法安装。

### 重现步骤

root@PandoraBox_5B5A:/root]#opkg print-architecture
arch all 1
arch noarch 1
arch mipsel_24kec_dsp 10

下载了 https://github.com/messense/aliyundrive-webdav/releases/download/v1.1.1/aliyundrive-webdav_1.1.1_mipsel_24kc.ipk

安装时 
root@PandoraBox_5B5A:/root]#opkg install aliyundrive-webdav_1.1.1_mipsel_24kc.ipk 
Unknown package 'aliyundrive-webdav'.
Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.

### 版本

1.1.1

### 运行平台

Youku l1c

### 日志

[root@PandoraBox_5B5A:/root]#cat /proc/cpuinfo 
system type             : MediaTek MT7620
machine                 : Youku YK1
processor               : 0
cpu model               : MIPS 24KEc V5.0
BogoMIPS                : 385.02
wait instruction        : yes
microsecond timers      : yes
tlb_entries             : 32
extra interrupt vector  : yes
hardware watchpoint     : yes, count: 4, address/irw mask: [0x0ffc, 0x0ffc, 0x0ffb, 0x0ffb]
isa                     : mips1 mips2 mips32r2
ASEs implemented        : mips16 dsp
shadow register sets    : 1
kscratch registers      : 0
core                    : 0
VCED exceptions         : not available
VCEI exceptions         : not available

**Comment by messense** [comment, marked as answer]
`/etc/opkg.conf` 文件内容增加如下内容后再安装。

```
arch all 1
arch noarch 1
arch mipsel_24kec_dsp 10
arch mipsel_24kc 100
```
